src/model/trust.page.model.ts

```diff
@@ -15,6 +15,7 @@ type TrustDetailsForm = {
 
 type TrustBeneficialOwnerListItem = {
   id: string;
+  ch_reference?: string
```

Comment:
consider passing a boolean is_newly_added instead of the ch_reference. The page should not need the actual hashed ids.